integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLiftAndShiftFromOnPremDomain.java

```diff
@@ -142,9 +148,11 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
     // install and verify operator
     installAndVerifyOperator(opNamespace, domainNamespace);
 
-    // install and verify Traefik
-    logger.info("Installing Traefik controller using helm");
-    traefikHelmParams = installAndVerifyTraefik(traefikNamespace, 0, 0);
+    if (!OKD) {
+      // install and verify Traefik
+      logger.info("Installing Traefik controller using helm");
+      traefikHelmParams = installAndVerifyTraefik(traefikNamespace, 0, 0);
+    }
```

Comment:
What about else condition  when running on OKD ?

Reply:
Same for all tests which installs LB'r in initAll(). Nothing to do here when running in OKD. We will create a route after the domain is up.